applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -575,68 +597,97 @@ void UPwSmallStrainInterfaceElement<TDim,TNumNodes>::
                                  const ProcessInfo& rCurrentProcessInfo)
 {
     KRATOS_TRY;
+
+    const GeometryType& rGeom = this->GetGeometry();
+    const unsigned int NumGPoints = rGeom.IntegrationPointsNumber(mThisIntegrationMethod);
+    std::vector<double> GPValues(NumGPoints);
+
+    //Printed on standard GiD Gauss points
+    const unsigned int OutputGPoints = rGeom.IntegrationPointsNumber(this->GetIntegrationMethod());
+    if (rValues.size() != OutputGPoints)
+        rValues.resize(OutputGPoints);
+
     // KRATOS_INFO("0-UPwSmallStrainInterfaceElement:::CalculateOnIntegrationPoints<double>()") << std::endl;
```

Comment:
```suggestion
  
```

Reply:
I've removed that line in my latest commit

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/10597/commits/d5414e8f08012988041b7a57f61ac353a4f1e518